docs/conf.py

```diff
@@ -64,7 +64,7 @@
 #
 import lasagne
 # The short X.Y version.
-version = '.'.join(lasagne.__version__.split('.', 3)[:2]).rstrip('dev')
+version = '.'.join(lasagne.__version__.split('.', 2)[:2])
```

Comment:
Just noticed that the variant before handled version strings we didn't even have, such as `0.1dev`. Simplified it to just keep the X.Y version no matter what. I'm not really sure if `version` is used anywhere in the templates we use, most things seem to use `release` which is the full version string.